### ISSUE TITLE
Web Service Sender does not clean up temp files

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3309]
Similar to MIRTH-2555. When the connector gets stopped or halted, any temp files it was using should be deleted. Right now the deletion only happens on JVM exit, even though the dispatch containers are not reused. So the number of cached WSDLs in the temp folder can grow essentially without bounds.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3408
Reporter: narupley
Created: 2014-08-15T17:45:23.000-0700

**Comment by rbeckman-nextgen** [comment]
Revision 7293: Temporary files are now correctly being cleaned up when Web Service Senders are stopped or halted.

Imported Comment. Original Details:
Author: narupley
Created: 2014-08-15T17:53:32.000-0700

**Comment by rbeckman-nextgen** [comment]
Was able to replicate this in an older build by creating a simple channel with a web service listener and simple authentication. I created another channel with a web service sender to dispatch messages to the listener channel. After deploying both channels and sending messages through, the wsdl was cached to the temp folder. Despite closing down the channel (stopping/undeploying) the wsdl remained until I stopped the service.

These channels were imported into 3.1 where I performed the same tests. This time, however, the wsdls were deleted once the channel was stopped as it should be.

Imported Comment. Original Details:
Author: jaysenp
Created: 2014-09-03T09:36:28.000-0700
